Beeper/BarcelonaMautrixIPC/Runtime/MautrixIPCChannel.swift

```diff
@@ -27,15 +38,16 @@ public class MautrixIPCChannel {
     
     var pongedOnce = false
     
-    private var cancellables = Set<OpenCombine.AnyCancellable>()
+    private var openCombineCancellables = Set<OpenCombine.AnyCancellable>()
```

Comment:
If these cancellable sets are just for storing, could we just make one set of cancellables that is a `Set<Any>` and store them all in there?

Reply:
I decided to refactor the serialization and parsing later, which will get rid of both eventually.